src/app/components/common/walletToggle/index.tsx

```diff
@@ -0,0 +1,39 @@
+import { selectAccount, selectIsConnectingToWallet } from "app/containers/BlockChain/Web3/selectors"
+import { Web3Actions } from "app/containers/BlockChain/Web3/slice"
+import { ConnectorPayload } from "app/containers/BlockChain/Web3/types"
+import { translations } from "locales/i18n"
+import { useEffect } from "react"
+import { useTranslation } from "react-i18next"
+import { useDispatch, useSelector } from "react-redux"
+import { LocalStorageKeys, storage } from "store/storage"
+import { ContainedButton } from "../buttons/containedButton"
+export const WalletToggle=()=>{
+  const { t } = useTranslation();
+  const dispatch = useDispatch()
+  const isConnecting = useSelector(selectIsConnectingToWallet)
+  const account = useSelector(selectAccount)
+
```

Comment:
this space is not needed since all are const, but what needs space is line 16-17